iceoryx_hoofs/include/iceoryx_hoofs/internal/cxx/optional.inl

```diff
@@ -107,10 +112,15 @@ inline optional<T>& optional<T>::operator=(const optional& rhs) noexcept
         {
             construct_value(rhs.value());
         }
+        else
+        {
+            // do nothing
+        }
     }
     return *this;
 }
 
+// AXIVION Next Construct AutosarC++19_03-A13.3.1 : It's safe and constrained by SFINAE
```

Comment:
Same as above, I also think it is a false positive. If so, the justification should be adapted.

Reply:
as addressed above